server/controllers/controllers.games.js

```diff
@@ -65,10 +66,28 @@ const postCommentByReviewId = (req, res, next) => {
     })
 }
 
+const patchReviewVotesById = (req, res, next) => {
```

Comment:
Have a look into how we are handling the error here for invalid ID. SQL should bounce back an error here if the id doesn't exist as it's a foreign key. So we shouldn't need to check to see if it exists first.
Or, if it doesn't automatically do this. We can check what is RETURNING and base the error on that.